graviti/dataframe/frame.py

```diff
@@ -718,8 +716,7 @@ def _process_record(
     ) -> Tuple[Callable[[Any], Any], bool]:
         processors: Dict[str, Callable[[Any], Any]] = {}
         need_process, sub_need_process = False, False
-        fields = schema.to_builtin().fields
-        for name, field in fields.items():
+        for name, field in schema.items():
```

Comment:
do we need to change the type hint of schema in the signature of `_process_record`?